File: samples/Test_JS/Resources/js/Control/WebViewTest.js
/**
 * Created by zhanglei on 16/8/4.
 */
var WebViewTest = ca.CAViewController.extend({
    ctor: function () {
        this._super();

        this.getView().setColor(ca.color._getGray());

        var web = ca.CAWebView.createWithLayout(DLayoutFill);
        web.loadURL("http://crossapp.9miao.com");
        this.getView().addSubview(web);
    },
    btncallback: function () {
        log("btncallback-->");
    }
});

File: samples/Test/Classes/Control/WebViewTest.cpp

#include "WebViewTest.h"

WebViewTest::WebViewTest()
{
    this->setTitle("CAWebView");
}

WebViewTest::~WebViewTest()
{
}

void WebViewTest::viewDidLoad()
{
    this->getView()->setColor(CAColor4B::GRAY);
    
    CAWebView* web = CAWebView::createWithLayout(DLayoutFill);
    web->loadURL("http://www.baidu.com");
    this->getView()->addSubview(web);
}

void WebViewTest::viewDidUnload()
{
    // Release any retained subviews of the main view.
    // e.g. self.myOutlet = nil;
}

